redfish-core/lib/log_services.hpp

```diff
@@ -1685,9 +1728,12 @@ inline void requestRoutesDBusEventLogEntry(App& app)
                         std::time_t timestamp{};
                         std::time_t updateTimestamp{};
                         std::string* severity = nullptr;
-                        std::string* message = nullptr;
                         std::string* filePath = nullptr;
+                        std::string* resolution = nullptr;
+                        std::string* eventId = nullptr;
+                        std::string* subsystem = nullptr;
                         bool resolved = false;
+                        bool serviceProviderNotify = false;
 
                         for (auto& propertyMap : resp)
```

Comment:
make this one const too.

Reply:
as responded previously closing this comment. I can address it in a separate commit as there is a bulk of code from @sbteeks, I will wait on that and do global change